paper-typeahead-input.html

```diff
@@ -449,6 +450,14 @@
         // }
       }
     },
+    _candidatesUpdated:function(candidates){
+      if(this.inputValue != undefined){
```

Comment:
typeof fails if the inputValue is null
[http://jsbin.com/lebaguk/1/edit?js,console](http://jsbin.com/lebaguk/1/edit?js,console)